### PR TITLE
Add functionality to logout OpenSRP and clear keyCloak session.

### PR DESCRIPTION
**Pull request by ciremusyoka**
Addresses https://github.com/onaio/reveal-frontend/issues/1302

Add the following:
- Functiaonality to logout of opensrp and clear browser keycloak session
- Catch errors introduced by unexpected token request response
- On unexpected token request response logout of opensrp and clear keaycloak session

### CODE REVIEW

**Review by moshthepitt — COMMENTED** [review]
No tests?

**Comment by ciremusyoka** [comment]
> No tests?

working on them

File: src/app/tests/index.test.ts [hunk 1]
```diff
@@ -219,6 +227,50 @@ describe('src/index.ts', () => {
...
+                panic(err, done);
+                expect(res.header.location).toEqual(`${EXPRESS_KEYCLOAK_LOGOUT_URL}?redirect_uri=${EXPRESS_SERVER_LOGOUT_URL}`);
+                expect(res.redirect).toBeTruthy();
+                expect(fetch).toHaveBeenCalledTimes(1);
+                expect(fetch).toHaveBeenCalledWith(
+                    EXPRESS_OPENSRP_LOGOUT_URL,
+                    {
+                        "headers": {
+                            "accept": "application/json",
+                            "authorization": "Bearer 64dc9918-fa1c-435d-9a97-ddb4aa1a8316",
```
Comment (moshthepitt) [inline src/app/tests/index.test.ts:265]: Where does the value of `64dc9918-fa1c-435d-9a97-ddb4aa1a8316` come from?

Comment (ciremusyoka) [reply src/app/tests/index.test.ts:265]: It is the access token in fixtures.

**Review by moshthepitt — APPROVED** [review]

**Pull request merged**